charts/controller/templates/ingress.yaml

```diff
@@ -0,0 +1,22 @@
+{{ if .Values.ingress.enabled - }}
```

Comment:
`{{ if .Values.ingress.enabled -}}`
the `-` should be next to the closing brackets. Currently results in a parse error of the template